### PR TITLE
Add some notes on scheduler groups

### PR DESCRIPTION
**Pull request by tmotyl**

### CODE REVIEW

File: doc/cron-groups.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Aoe_Scheduler is capable of splitting you crons into different groups so you can run them in parallel or distribute them on multiple servers.
+Running multiple scheduler jobs in parallel is quite handy when you have some jobs which take a long time to run, and are not crucial for shop to run properly. En example could be a job which generates some kind of reports.
```
Comment (LeeSaferite) [inline doc/cron-groups.md]: "An example"

File: doc/cron-groups.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Aoe_Scheduler is capable of splitting you crons into different groups so you can run them in parallel or distribute them on multiple servers.
```
Comment (LeeSaferite) [inline doc/cron-groups.md]: "you crons" -> "your cron jobs"

File: doc/cron-groups.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                </run>
+                <groups>reporting,other_group_name</groups>
+            </job_name>
+        </jobs>
+    </crontab>
+    ```
+    Aoe_Scheduler will merge xml configuration with the one from db.
+
+2. Configure your crontab entry using `--includeGroups` and `--excludeGroups`
+You can find examples in `System > Scheduler > Instructions`. See more about [Instructions here](doc/instructions.md)
```
Comment (hostep) [inline doc/cron-groups.md]: Small remark: the link `doc/instructions.md` should be changed to `instructions.md`, otherwise you'll have a broken link.

**Comment by tmotyl** [comment]
thanks @hostep ! I will push a followup

**Comment by tmotyl** [comment]
fixed in https://github.com/AOEpeople/Aoe_Scheduler/pull/245

**Pull request merged**